apps/docusign/src/app/api/webhooks/elba/users/delete/route.ts

```diff
@@ -0,0 +1,13 @@
+import type { NextRequest } from 'next/server';
+import { deleteUsers } from './service';
+
+type RequestParams= {
+  userId: string;
+}
+export async function POST(request: NextRequest) {
+  const data: RequestParams = await request.json() as RequestParams
```

Comment:
```suggestion
  const data: unknown = await request.json() as RequestParams
  const { userId } = parseWebhookEventData(
    'users.delete_user_requested,
    data
  );
```
You have to check if `users.delete_user_requested` is implemented on your branch, if it's not the case you will have to rebase on staging.